docs/Usage-Guide.md

```diff
@@ -1,3 +1,172 @@
-# Usage Guide
-
----Organize the Usage Guide into sections using Markdown Headings. This guide should mirror the steps in the Quick Start but with far more details and include any optional steps.---
+# Usage Guide
+
+1. Download the package from the repository [Releases](https://github.com/Kentico/xperience-by-kentico-sitefinity-migration-tool/releases)
+
+   - Optionally, clone the git repository and build from the source files.
+   - ` git clone -v "https://github.com/Kentico/xperience-by-kentico-sitecore-migration-tool.git" "{YOUR PATH}"`
+
+2. Modify the console application's configuration file `/examples/Migration.Toolkit.Sitefinity.Console/appsettings.json`
+   - The details for configuration options can be found below.
+3. Modify code for any additional customizations
+   - Entity Framework is included. Add additional custom models to Migration.Toolkit.Sitefinity.Data.Core.EF.SitefinityContext for any custom tables. An additional import service will need to be created. See other import services.

```

Comment:
See other import services where?

Reply:
This has been resolved. I have added a path to the other import services so the reader can check the code.